conanfile.py

```diff
@@ -19,29 +19,32 @@ class Recipe(ConanFile):
 
     # Binary configuration
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "fPIC": [True, False]}
-    default_options = {"shared": False, "fPIC": True}
+    options = {"shared": [True, False], "fPIC": [True, False], "with_test_deps": [True, False]}
+    default_options = {"shared": False, "fPIC": True, "with_test_deps": False}
     exports = "LICENSE",
     exports_sources = "src/*", "private/*", "CMakeLists.txt", "cmake/*"
-    requires = (("fmt/9.0.0", "private"),  # format must only be used within cpp files
```

Comment:
where did fmt go?

Reply:
I intentionally removed with the rationale being that I'm trying to shave compile time wherever possible in our projects and `fmt` was very rarely used. Additionally all cases where it was could trivially be replaced by things in `std` without much downside. So it was kind of unecessary to have this as an additional dependency.

But of course if you feel that we absolutely need it I can revert those changes.

Reply:
OK, I am convinced.